tools/noise/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+project(noise-tools C)
+
+find_library(M_LIBRARY m)
+
+add_executable(dt-noiseprofile noiseprofile.c)
+target_link_libraries(dt-noiseprofile ${M_LIBRARY})
+
+install(TARGETS
+            dt-noiseprofile
+        DESTINATION
+            ${CMAKE_INSTALL_LIBEXECDIR}/darktable/tools
+        COMPONENT
+            DTApplication)
+
+if (UNIX AND NOT WIN32)
```

Comment:
I guess we can move the whole content of the file into this check. Why would we install a helper tool when the main program is missing?
That being said, `UNIX AND NOT WIN32` sounds wrong.

Reply:
Nope, MinGW is UNIX and WIN32 ;-)